packages/ui/src/components/address-displayer/address-displayer.native.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { BtcAvatarIcon } from '../avatar/btc-avatar-icon.native';
+import { StxAvatarIcon } from '../avatar/stx-avatar-icon.native';
+import { Box } from '../box/box.native';
+import { Flag } from '../flag/flag.native';
+import { AddressDisplayer as Component } from './address-displayer.native';
+
+const meta: Meta<typeof Component> = {
+  component: Component,
+  title: 'AddressDisplayer',
+};
+
+export default meta;
+type Story = StoryObj<typeof Component>;
+
+export const Bitcoin: Story = {
+  args: {
+    address: 'bc1pmzfrwwndsqmk5yh69yjr5lfgfg4ev8c0tsc06e',
+  },
+};
+
+export const Stacks: Story = {
+  args: {
+    address: 'SP3XKZE32KZ925AAAGWPG1W66YP5BM2RD73T6AJHS',
+  },
+};
+
+export const WithBTCIcon: Story = {
```

Comment:
Typ we've been using `Btc` and `Stx` in naming.

Reply:
https://www.notion.so/trustmachines/Naming-Camelcase-acronyms-119d74b694c78080974aee5e3bc18bc9?pvs=4